tests/support.bash

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+export SCRIPT_DIR
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/.."
+
+# the shell to test: driven by CI
+export TEST_SHELL=${TEST_SHELL:-bash}
+export TEST_CONNECTION=${TEST_CONNECTION:-lcl}
+
+export OS=
+case "$(uname)" in
+    Darwin) OS=osx ;;
+    Linux)  OS=linux ;;
+    *)      OS=unknown ;;
+esac
+
+function setup
+{
+    export TMP_DIR="$BATS_TMPDIR/liquidprompt-tests"
+    rm -rf "$TMP_DIR"
+    mkdir -p "$TMP_DIR"
+
+    # on OSX, we are handed a deep path to some
+    # /var/private/tmp/ directory
```

Comment:
Because path shortening would yield different results based on platform. I will add to the comment